[2025-03-14T20:45:03.567+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T20:45:03.656+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_weather_etl.load scheduled__2025-03-14T20:40:00+00:00 [queued]>
[2025-03-14T20:45:03.660+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_weather_etl.load scheduled__2025-03-14T20:40:00+00:00 [queued]>
[2025-03-14T20:45:03.660+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-14T20:45:03.664+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load> on 2025-03-14 20:40:00+00:00
[2025-03-14T20:45:03.667+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1249) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-14T20:45:03.669+0000] {standard_task_runner.py:72} INFO - Started process 1250 to run task
[2025-03-14T20:45:03.669+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'open_weather_etl', 'load', 'scheduled__2025-03-14T20:40:00+00:00', '--job-id', '190', '--raw', '--subdir', 'DAGS_FOLDER/open_weather_dag.py', '--cfg-path', '/tmp/tmpo5fli6_u']
[2025-03-14T20:45:03.669+0000] {standard_task_runner.py:105} INFO - Job 190: Subtask load
[2025-03-14T20:45:03.689+0000] {task_command.py:467} INFO - Running <TaskInstance: open_weather_etl.load scheduled__2025-03-14T20:40:00+00:00 [running]> on host c20e6545b232
[2025-03-14T20:45:03.721+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='lherrera5034@gmail.com' AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='open_weather_etl' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T20:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-14T20:40:00+00:00'
[2025-03-14T20:45:03.721+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T20:45:03.728+0000] {logging_mixin.py:190} INFO - Data Loaded into Postgres database
[2025-03-14T20:45:03.728+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-14T20:45:03.732+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T20:45:03.733+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=open_weather_etl, task_id=load, run_id=scheduled__2025-03-14T20:40:00+00:00, execution_date=20250314T204000, start_date=20250314T204503, end_date=20250314T204503
[2025-03-14T20:45:03.769+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T20:45:03.776+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-14T20:45:03.777+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
